docs/designs/xref.md

```diff
@@ -91,8 +91,42 @@ xref:
   - http://url1/.xrefmap.json
   - http://url2/.xrefmap.json
 ```
+User should be able to set a string instead of array, docfx will expand it to an array during the build.\
 During `docfx restore`, all the JSON files will be restored and merged.
 
+#### Xref base path
+Since we do not have a clear picture how we are going to support the local build, we will focus on OPS build firstly.\
+Unlike DocFX v2, the user of v3 needs to set where to get the `.xrefmap.json` files in `docfx.yml`.
+```yml
+xrefBasePath:
```

Comment:
Is this first URL segment or docset base path? because base path can contain multiple URL segments like `uwp/api`

Reply:
we found that the user of docfx v2 always needs to config this in `xref_query_tags` under `.openpublishing.publish.config.json`, updating the doc